magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -185,8 +185,8 @@ create_irf:
 
     energy_bins:
         start: 0.01   # unit: [TeV]
-        stop: 100   # unit: [TeV]
-        n_bins: 20   # number of bins in log space
+        stop: 1000   # unit: [TeV]
```

Comment:
I increased it here, because I had crashes with events going outside of energy range, but we might want to have a different type of fix instead - such that the narrower binning than MC range can be used

Reply:
So far I keep this change, but would be good to know what kind of crashes happened